src/__tests__/ShieldhitConverter.test.ts

```diff
@@ -1,6 +1,14 @@
-import { readFileSync } from 'fs';
 import { Builder, By, until, WebDriver } from 'selenium-webdriver';
 import chrome from 'selenium-webdriver/chrome';
+import expectedBeamText from 'src/libs/converter/tests/shieldhit/resources/expected_shieldhit_output/beam.dat?raw';
+import expectedDetectText from 'src/libs/converter/tests/shieldhit/resources/expected_shieldhit_output/detect.dat?raw';
+import expectedGeoText from 'src/libs/converter/tests/shieldhit/resources/expected_shieldhit_output/geo.dat?raw';
+import expectedMatText from 'src/libs/converter/tests/shieldhit/resources/expected_shieldhit_output/mat.dat?raw';
+
+// In expected output file we have characters that do not appear in the file generated by simulator, they should be removed to get truth in comparison
```

Comment:
Mention these characters here in the comment, its hard to guess from regexp what is that ?
I see some whitespace, but what else ?